src/Core/Checkout/Cart/Order/RecalculationService.php

```
@@ -133,18 +141,10 @@ public function addProductToOrder(string $orderId, string $productId, int $quant
             $this->addProductToDeliveryPosition($new, $recalculatedCart);
         }
 
-        $conversionContext = $this->getOrderConversionContext();
-
+        $conversionContext = $this->getOrderConversionContext()->setIncludeDeliveries(true);
```

Same here?

---

same as above, but this time it's explicitly `true` as we added a line item and therefore it's ensured to have one - at least we don't want to hide any error here